contracts/DandelionOrg.sol

```diff
@@ -92,28 +91,41 @@ contract DandelionOrg is BaseTemplate {
     * @param _timeLockSettings Array of [_lockDuration, _lockAmount, _spamPenaltyFactor] to set up the timeLock app of the organization
     * @param _votingSettings Array of [supportRequired, minAcceptanceQuorum, voteDuration] to set up the voting app of the organization
     * @param _executionDelay execution delay time to set up the delay app
+    * @param _dissentWindowBlocks window block for the dissent oracle
     */
     function installDandelionApps(
         string _id,
         address[] _redemptionsRedeemableTokens,
         address[] _tokenRequestAcceptedDepositTokens,
         address _timeLockToken,
         uint256[3] _timeLockSettings,
-        uint64[3] _votingSettings,
-        uint64 _executionDelay
+        uint64[4] _votingSettings,
+        uint64 _executionDelay,
+        uint64 _dissentWindowBlocks
     )
         external
     {
+        _ensureDandelionSettings(_tokenRequestAcceptedDepositTokens, _timeLockToken, _timeLockSettings, _votingSettings);
         _ensureBaseAppsCache();
         Kernel dao = _popDaoCache();
         ACL acl = ACL(dao.acl());
         bool agentAsVault = _popAgentAsVaultCache();
 
-        _installDandelionApps(dao, _redemptionsRedeemableTokens, _tokenRequestAcceptedDepositTokens, _timeLockToken, _timeLockSettings, _votingSettings, _executionDelay);
-        (Voting dandelionVoting,,,,,) = _popDandelionAppsCache();
+        DandelionVoting dandelionVoting = _installDandelionVotingApp(dao, _votingSettings);
+        _setupBasePermissions(acl, agentAsVault, dandelionVoting);
```

Comment:
I think this 2 functions  could go inside `_installDandelionApps`. Also instead of `dandelionVoting` being the grantee, it should be the `delay` App

Reply:
i did it that way so we don't need to cache the voting app after installing, so that way can be used right after for setup the base permissions and for the _transferRootPermissionsFromTemplateAndFinalizeDAO

Reply:
oh but yes you are right to setup the base permissions we need the delay not the voting